### ISSUE TITLE
k8s.io/api/authentication/v1/zz_generated.deepcopy.go:61: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no fieldor method DeepCopyInto

### ISSUE THREAD

**Issue by huangjiasingle** [kubernetes/api#3]
the code of master can't build success! it's some error like : 

```
../../project/src/k8s.io/api/authentication/v1/zz_generated.deepcopy.go:61: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no fieldor method DeepCopyInto)
# k8s.io/api/certificates/v1beta1
../../project/src/k8s.io/api/certificates/v1beta1/zz_generated.deepcopy.go:65: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/certificates/v1beta1/zz_generated.deepcopy.go:93: in.LastUpdateTime.DeepCopyInto undefined (type v1.Time has no field or method DeepCopyInto)
# k8s.io/api/authorization/v1beta1
../../project/src/k8s.io/api/authorization/v1beta1/zz_generated.deepcopy.go:77: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/authorization/v1beta1/zz_generated.deepcopy.go:138: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/authorization/v1beta1/zz_generated.deepcopy.go:201: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
# k8s.io/api/admissionregistration/v1alpha1
../../project/src/k8s.io/api/admissionregistration/v1alpha1/zz_generated.deepcopy.go:140: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/admissionregistration/v1alpha1/zz_generated.deepcopy.go:240: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
# k8s.io/api/authentication/v1beta1
../../project/src/k8s.io/api/authentication/v1beta1/zz_generated.deepcopy.go:61: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
# k8s.io/api/storage/v1
../../project/src/k8s.io/api/storage/v1/zz_generated.deepcopy.go:53: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
# k8s.io/api/rbac/v1alpha1
../../project/src/k8s.io/api/rbac/v1alpha1/zz_generated.deepcopy.go:89: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/rbac/v1alpha1/zz_generated.deepcopy.go:123: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field ormethod DeepCopyInto)
../../project/src/k8s.io/api/rbac/v1alpha1/zz_generated.deepcopy.go:265: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field ormethod DeepCopyInto)
../../project/src/k8s.io/api/rbac/v1alpha1/zz_generated.deepcopy.go:299: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field ormethod DeepCopyInto)
# k8s.io/api/scheduling/v1alpha1
../../project/src/k8s.io/api/scheduling/v1alpha1/zz_generated.deepcopy.go:53: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
# k8s.io/client-go/tools/clientcmd/api
../../project/src/k8s.io/client-go/tools/clientcmd/api/zz_generated.deepcopy.go:111: val.DeepCopyObject undefined (type runtime.Object has no field or method DeepCopyObject)
../../project/src/k8s.io/client-go/tools/clientcmd/api/zz_generated.deepcopy.go:166: val.DeepCopyObject undefined (type runtime.Object has no field or method DeepCopyObject)
../../project/src/k8s.io/client-go/tools/clientcmd/api/zz_generated.deepcopy.go:230: val.DeepCopyObject undefined (type runtime.Object has no field or method DeepCopyObject)
../../project/src/k8s.io/client-go/tools/clientcmd/api/zz_generated.deepcopy.go:266: val.DeepCopyObject undefined (type runtime.Object has no field or method DeepCopyObject)
../../project/src/k8s.io/client-go/tools/clientcmd/api/zz_generated.deepcopy.go:293: val.DeepCopyObject undefined (type runtime.Object has no field or method DeepCopyObject)
# k8s.io/api/policy/v1beta1
../../project/src/k8s.io/api/policy/v1beta1/zz_generated.deepcopy.go:67: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field ormethod DeepCopyInto)
../../project/src/k8s.io/api/policy/v1beta1/zz_generated.deepcopy.go:74: (*in).DeepCopyInto undefined (type *v1.DeleteOptions has no field or method DeepCopyInto)
../../project/src/k8s.io/api/policy/v1beta1/zz_generated.deepcopy.go:103: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/policy/v1beta1/zz_generated.deepcopy.go:180: (*in).DeepCopyInto undefined (type *v1.LabelSelector has no field or method DeepCopyInto)
../../project/src/k8s.io/api/policy/v1beta1/zz_generated.deepcopy.go:212: invalid indirect of val.DeepCopy() (type v1.Time)
# k8s.io/api/authorization/v1
../../project/src/k8s.io/api/authorization/v1/zz_generated.deepcopy.go:77: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/authorization/v1/zz_generated.deepcopy.go:138: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no fieldor method DeepCopyInto)
../../project/src/k8s.io/api/authorization/v1/zz_generated.deepcopy.go:201: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no fieldor method DeepCopyInto)
# k8s.io/api/storage/v1beta1
../../project/src/k8s.io/api/storage/v1beta1/zz_generated.deepcopy.go:53: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
# k8s.io/api/rbac/v1beta1
../../project/src/k8s.io/api/rbac/v1beta1/zz_generated.deepcopy.go:89: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/rbac/v1beta1/zz_generated.deepcopy.go:123: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/rbac/v1beta1/zz_generated.deepcopy.go:265: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/rbac/v1beta1/zz_generated.deepcopy.go:299: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
# k8s.io/api/core/v1
../../project/src/k8s.io/api/core/v1/zz_generated.deepcopy.go:897: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/core/v1/zz_generated.deepcopy.go:1013: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/core/v1/zz_generated.deepcopy.go:1079: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/core/v1/zz_generated.deepcopy.go:1440: in.StartedAt.DeepCopyInto undefined (type v1.Time has no field or method DeepCopyInto)
../../project/src/k8s.io/api/core/v1/zz_generated.deepcopy.go:1457: in.StartedAt.DeepCopyInto undefined (type v1.Time has no field or method DeepCopyInto)
../../project/src/k8s.io/api/core/v1/zz_generated.deepcopy.go:1458: in.FinishedAt.DeepCopyInto undefined (type v1.Time has no field or method DeepCopyInto)
../../project/src/k8s.io/api/core/v1/zz_generated.deepcopy.go:1788: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/core/v1/zz_generated.deepcopy.go:1967: in.ObjectMeta.DeepCopyInto undefined (type v1.ObjectMeta has no field or method DeepCopyInto)
../../project/src/k8s.io/api/core/v1/zz_generated.deepcopy.go:1970: in.FirstTimestamp.DeepCopyInto undefined (type v1.Time has no field or methodDeepCopyInto)
../../project/src/k8s.io/api/core/v1/zz_generated.deepcopy.go:1971: in.LastTimestamp.DeepCopyInto undefined (type v1.Time has no field or method DeepCopyInto)
../../project/src/k8s.io/api/core/v1/zz_generated.deepcopy.go:1971: too many errors
```

**Comment by Winkies** [comment]
I had same issue. I didn't build it, just used the latest version of k8s.io/api (code of master) to parse yaml deployment file.

It's not a solution but it works for me : I checkout to [4b8fc5b](https://github.com/kubernetes/api/commit/4b8fc5be9b77d91bbb6525d18591c43699a2b4e5)

**Comment by ianvernon** [comment]
+1 .

**Comment by thirteenmans** [comment]
I had same issue. And I find this may depend on apimachinery version.
I used as below:
cd apimachinery && git checkout release-1.7
cd api && git checkout master
go build ./...
It will get the same issue.

Then I tried the following combination：
apimachinery: master && api: master
apimachinery: release-1.9 && api: master
apimachinery: release-1.8 && api: master
They are all can build successfully！

So, I suggest that try to use apimachinery version >= release-1.8 with api.
